lib/screens/app/app.dart

```diff
@@ -114,55 +127,137 @@ class _AppState extends State<App> with WidgetsBindingObserver {
       case AppLifecycleState.paused:
         break;
       case AppLifecycleState.resumed:
-        Provider.of<ConnectionNotifier>(context, listen: false).discoverEndpoints();
+        Provider.of<ConnectionNotifier>(context, listen: false)
+            .discoverEndpoints();
         break;
       case AppLifecycleState.detached:
         break;
     }
   }
 
+  void processIdentityRequest(request) async {
+    if (request.manager.data.req[1] == null ||
+        request.manager.data.req[1]["permission"] == null) {
+      request.manager.data.req[1]["permission"] = {
+        "actor": SettingsNotifier.of(context, listen: false).accountName,
+        "permission": "active"
+      };
+    }
+
+    var confirmed = await showDialog(
+        context: context,
+        builder: (_) {
+          return AlertDialog(
+            title: Text("Confirm Authorization"),
+            actions: [
+              FlatButton(
+                child: Text("Cancel"),
+                onPressed: () => Navigator.of(context).pop(),
+              ),
+              FlatButton(
+                child: Text("Confirm"),
+                onPressed: () => Navigator.of(context).pop(true),
+              ),
+            ],
+          );
+        });
+
+    if (confirmed == true) {
+      try {
+        var abis = await request.manager.fetchAbis();
+
+        var signer = Authorization()
+          ..actor = SettingsNotifier.of(context, listen: false).accountName
+          ..permission = "active";
+
+        ResolvedSigningRequest resolved =
+            request.manager.resolve(abis, signer, null);
+
+        var signBuf = Uint8List.fromList(List.from(ser.stringToHex(chainId))
+          ..addAll(resolved.serializedTransaction)
+          ..addAll(Uint8List(32)));
+
+        var walletPrivateKey =
+            SettingsNotifier.of(context, listen: false).privateKey;
+
+        var signature =
+            EOSPrivateKey.fromString(walletPrivateKey).sign(signBuf).toString();
+
+        var transactionId = resolved.getTransactionId().toLowerCase();
+        var body = """{
+                "tx": "$transactionId",
+                "sig": "$signature", 
+                "rbn": "${resolved.transaction.refBlockNum.toString()}",
+                "rid": "${resolved.transaction.refBlockPrefix.toString()}",
+                "ex": "${resolved.transaction.expiration.toString()}",
+                "req": "${resolved.request.encode()}",
+                "sa": "${resolved.signer.actor}",
+                "sp": "${resolved.signer.permission}",
+                "cid": "$chainId"
+              }""";
+
+        await post(
+          request.manager.data.callback,
+          body: body,
+        );
+      } catch (err) {
+        print(err);
+      }
+    }
+  }
+
+  void processTransactionRequest(request) async {
```

Comment:
should not live here. Move to appropriate class/file

Reply:
anchor_auth_service.dart ?

Something like that. @7flash